zsh-ai-commands.zsh

```diff
@@ -18,79 +20,103 @@ fzf_ai_commands() {
 
   [ -n "$BUFFER" ] || { echo "Empty prompt" ; return }
 
+  BUFFER="$(echo "$BUFFER" | sed 's/^AI_ASK: //g')"
+
   ZSH_AI_COMMANDS_USER_QUERY=$BUFFER
 
+  # save to history
+  echo "AI_ASK: $ZSH_AI_COMMANDS_USER_QUERY" >> $HISTFILE
+  # also to atuin's history if installed
+  if command -v atuin &> /dev/null;
+  then
+      atuin_id=$(atuin history start "AI_ASK: $ZSH_AI_COMMANDS_USER_QUERY")
+      atuin history end --exit 0 "$atuin_id"
+  fi
```

Comment:
Personally I think the history is the wrong place for natural language prompts. If it is ok with you I will place this feature behind a feature flag, which I would like to set to false by default.